fiware/CMakeLists.txt

```diff
@@ -20,13 +20,30 @@ project(soss-fiware)
 
 find_package(soss-core REQUIRED)
 
+if(WIN32)
+    find_package(yaml-cpp REQUIRED)
+    include(FindCURL)
+    find_package(curl REQUIRED)
+endif()
+
 ########################################################################
 # Find curl
-find_path(CURLPP_DIR NAMES "include/curlpp/cURLpp.hpp")
+find_path(CURLPP_DIR NAMES "include/curlpp/cURLpp.hpp" CMAKE_FIND_ROOT_PATH_BOTH)
```

Comment:
For curiosity, what CMAKE_FIND_ROOT_PATH_BOTH does?

Reply:
Based on https://cmake.org/cmake/help/v3.8/command/find_path.html?highlight=i it makes find_path to use a specific search in different paths it has. In windows the path CURLPP_DIR is only found whit that option.